### PR TITLE
Quick fixes

### PR DESCRIPTION
**Pull request by Janrupf**
some really small fixes for issues discovered while working on the java wrapper

### CODE REVIEW

File: src/render_engine/vulkan/vulkan_render_engine_init.cpp [hunk 1]
```diff
@@ -10,20 +10,18 @@
 #include "vulkan_utils.hpp"
 
 namespace nova::renderer {
-    vulkan_render_engine::vulkan_render_engine(const nova_settings& settings, nova::ttl::task_scheduler* task_scheduler)
-        : render_engine(settings, task_scheduler) {
+    vulkan_render_engine::vulkan_render_engine(const nova_settings& n_settings, nova::ttl::task_scheduler* task_scheduler)
```
Comment (DethRaid) [inline src/render_engine/vulkan/vulkan_render_engine_init.cpp]: Don't need to change the name. The compiler is able to disambiguate `settings` from `this->settings` in the constructor's initializer list

**Review by DethRaid — CHANGES_REQUESTED** [review]

**Comment by DethRaid** [comment]
Is this expected to fail CI?

**Comment by cwfitzgerald** [comment]
Yes. CI assumes vcpkg branch or a child thereof.

**Review by DethRaid — APPROVED** [review]

**Comment by cwfitzgerald** [comment]
Overriding tests due to broken tests, merging.

**Pull request merged**